modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/ProcessStore.java

```diff
@@ -458,7 +459,7 @@ public String createBPMN(String bpmnName, String bpmnVersion, Object o, String u
                 byte[] bpmnContent = IOUtils.toByteArray(bpmnStream);
                 String bpmnText = new String(bpmnContent);
                 bpmnContentResource.setContent(bpmnText);
-                bpmnContentResource.setMediaType("application/xml");
+                bpmnContentResource.setMediaType(MediaType.APPLICATION_XML);
                 String bpmnContentPath = "bpmncontent/" + bpmnName + "/" + bpmnVersion;
```

Comment:
Use **File.separator** instead of "/" every place possible. Because Linux uses forward slash and Windows uses back slash.